les/vflux/server/balance_tracker.go

```diff
@@ -40,7 +41,7 @@ type BalanceTrackerSetup struct {
 	PriorityFlag, UpdateFlag nodestate.Flags
 	BalanceField             nodestate.Field
 	// external connections
-	connAddressField, capacityField nodestate.Field
```

Comment:
I think my first suggestion to this big PR is: now we can lowercase these fields and flags!

Reply:
Or is there some elegant approaches we can get rid of the `Connect` function? Now all the components are in the same package.

Reply:
We can actually lowercase most of the package now :) I kept the connect function though because I use different test client types and corresponding test fields for priorityPool and balanceTracker tests which are wired together statically. Of course we can change it if you have a more elegant idea.